pkg/api/serialization_test.go

```diff
@@ -415,9 +416,15 @@ func TestSpecificKind(t *testing.T) {
 	}
 }
 
+// Keep this in sync with the respective upstream set
+var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions")
+
 func TestTypes(t *testing.T) {
 	for kind, reflectType := range api.Scheme.KnownTypes("") {
-		if !strings.Contains(reflectType.PkgPath(), "/origin/") {
+		if !strings.Contains(reflectType.PkgPath(), "/origin/") && reflectType.PkgPath() != "k8s.io/kubernetes/pkg/api" {
```

Comment:
I like the present name. How about adding godoc?

On Fri, Oct 30, 2015 at 3:10 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/api/serialization_test.go
> <https://github.com/openshift/origin/pull/5485#discussion_r43505303>:
>
> >  func TestTypes(t *testing.T) {
> >  	for kind, reflectType := range api.Scheme.KnownTypes("") {
> > -		if !strings.Contains(reflectType.PkgPath(), "/origin/") {
> > +		if !strings.Contains(reflectType.PkgPath(), "/origin/") && reflectType.PkgPath() != "k8s.io/kubernetes/pkg/api" {
>
> ah. Can we update the name of this test if the intent is to test
> serialized k8s types?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5485/files#r43505303>.
>